cloud-sql/mysql/client-side-encryption/requirements.txt

```diff
@@ -0,0 +1,3 @@
+SQLAlchemy==1.3.23
```

Comment:
I know a newer version of SQLAlchemy has come out since 1.3.23 - will the 1.4.x break you? If so, please lmk. We'll either add a comment about it here or add a constraints file (I'm in the middle of getting constraints file checking merged to the noxfile)